### PR TITLE
Task 9

### PR DESCRIPTION
**Pull request by GuilhermeGuimaraesDeMedeiros**
Solving Task 11.

### CODE REVIEW

**Review by flaviostutz — CHANGES_REQUESTED** [review]
Please delete the node_modules folder from the commit and add it to gitignore so git ignores it then. Normally we don't commit downloaded or generated files to the repo.

**Comment by GuilhermeGuimaraesDeMedeiros** [comment]
Trying to Solve Task 11....

File: task9/persons.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[
```
Comment (flaviostutz) [inline task9/persons.json:1]: This file shouldn't be committed as it's part of the runtime of the application running. Please remove it and add to gitignore too

**Review by flaviostutz — CHANGES_REQUESTED** [review]
Please move the application code to root folder or to another folder with a meaningful name because "task9" is not part of the application itself :)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+persons/
```
Comment (flaviostutz) [inline .gitignore:2]: Why did you included the dir instead of the file?

**Review by flaviostutz — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (GuilhermeGuimaraesDeMedeiros) [reply .gitignore]: My mistake....I will fix it.

Comment (GuilhermeGuimaraesDeMedeiros) [reply .gitignore]: Fixed!

**Review by flaviostutz — CHANGES_REQUESTED** [review]
Avoid using directories names with spaces or that are too long.
Pls move the files to the root directory of the repo to make things simpler.

**Comment by GuilhermeGuimaraesDeMedeiros** [comment]
Fixed

**Comment by GuilhermeGuimaraesDeMedeiros** [comment]
I think...rsrs

**Review by flaviostutz — APPROVED** [review]
Nice man!!!

**Pull request merged**